README.bot.md

```
@@ -0,0 +1,243 @@
+
+logging:
+  level: info
+  format: json
+  output: /var/log/lark-bot.log
+```
+
+---
+
+## 🏗️ 架构设计
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)